l10n_br_zip/tests/test_l10n_br_zip.py

```diff
@@ -0,0 +1,74 @@
+# @ 2019 Akretion - www.akretion.com.br -
+#   Magno Costa <magno.costa@akretion.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo.tests.common import TransactionCase
+
+
+class L10nBRZipTest(TransactionCase):
+
+    def setUp(self):
+        super(L10nBRZipTest, self).setUp()
+        self.zip_obj = self.env['l10n_br.zip']
+        self.zip_1 = self.zip_obj.create(dict(
+            zip='88032050',
+            city_id=self.env.ref('l10n_br_base.city_4205407').id,
+            state_id=self.env.ref('base.state_br_sc').id,
+            country_id=self.env.ref('base.br').id,
+            street='Donicia',
+            street_type='Rua',
+            district='centro',
+        ))
+        self.company = self.env.ref('base.main_company')
+        self.company_1 = self.env['res.company'].create(dict(
+            name='teste',
+            street='donicia',
+            country_id=self.env.ref('base.br').id,
+            state_id=self.env.ref('base.state_br_sc').id,
+            city_id=self.env.ref('l10n_br_base.city_4205407').id,
+        ))
+
+    def test_search_zip_code_company(self):
+        self.company.zip = '88032050'
+        self.company.zip_search()
+        self.assertEquals(
+            self.company.district, 'centro',
+            'Error in method zip_search to mapping field district.')
+        self.assertEquals(
+            self.company.street, 'Rua Donicia',
+            'Error in method zip_search to mapping field street.')
+        self.assertEquals(
+            self.company.city_id.name, u'Florianópolis',
```

Comment:
no more need for such unicode with Python 3

Reply:
@rvalyi @renatonlima  Fiz um PR pra gente conseguir dar andamento a isso.
https://github.com/akretion/l10n-brazil/pull/82